src/features/people/PopularPeoplePage/PopularPeople/index.js

```diff
@@ -0,0 +1,30 @@
+import { Container } from "../../../../common/Container";
+import TilesSection from "../../../../common/TilesSection";
+import ActorTile from "../../ActorTile/index";
+import { useSelector } from "react-redux";
+import { selectPeople } from "../../peopleSlice";
+import { GridList } from "../../../../common/GridList";
+
+const PopularPeople = () => {
+  const people = useSelector(selectPeople);
+
+  return (
+    <Container>
+      <TilesSection title="Popular people">
+        <GridList popularPeople>
+          {people &&
```

Comment:
with loading handled we can change `people && people.map...` to just `people.map...`